pkg/thing/interactors/errors.go

```diff
@@ -20,4 +20,22 @@ var (
 
 	// ErrSensorsNotProvided is returned when thing's sensors are not provided
 	ErrSensorsNotProvided = errors.New("thing's sensors not provided")
+
+	// ErrIDLength is returned when the thing's id have more than 16 ascii characters
+	ErrIDLength = errors.New("id length exceeds 16 characters")
+
+	// ErrIDNotInHex is returned when the thing's id is not formatted in hexadecimal base
+	ErrIDNotInHex = errors.New("id is not in hexadecimal format")
```

Comment:
`ErrIDNotHex` sounds better to me. What do you think, @lcbm and @esdrasjnr?

Reply:
Agreed @joaoaneto! I don't think we need to use prepositions when declaring identifiers :smile:

Reply:
Done!